ArjunaJTA/jms/src/test/java/org/jboss/narayana/jta/jms/JmsXAResourceRecoveryHelperTests.java

```diff
@@ -83,6 +84,15 @@ public void shouldCreateConnectionOnScanStart() throws Exception {
     }
 
     @Test
+    public void shouldCreateConnectionWithCredentialsOnScanStart() throws Exception {
+        when(xaConnectionFactoryMock.createXAConnection(anyString(), anyString())).thenReturn(xaConnectionMock);
```

Comment:
It was just a remark about the way Mockito does things. 

    if (xaConnectionFactoryMock.createXAConnection(anyString(), anyString()) {
        return(xaConnectionMock);
    }

is the way to go IMHO